js-weather/index.js

```diff
@@ -0,0 +1,50 @@
+var API_KEY = '4b5774e9f3d2a07b84f0f2f88e486224';
+var city = 'London';
+
+function ajax() {
```

Comment:
functionキーワードじゃなくても大丈夫な関数はアロー関数を使いましょう。
また、グローバル変数(今回は `city`)を書き換える仕様は不健全なので、
関数の引数にしましょう。つまりこうしましょう

```suggestion
const ajax = (city) => {
```

Reply:
3b93282

こちらのコミットで対応しました。
他の変更も同時にコミットしており申し訳ありません。以後注意します。